packages/backend/src/services/UserService.ts

```diff
@@ -1014,7 +1014,13 @@ export class UserService extends BaseService {
         const updatedUser = await this.userModel.updateUser(
             user.userUuid,
             user.email,
-            data,
+            {
```

Comment:
User routes are not using TSOA to validate body properties so I'm made sure it doesn't update properties it shouldn't. Related: https://github.com/lightdash/lightdash/issues/6879